app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -967,7 +967,11 @@ public function checkForDuplicateContact(array $queryFields, Lead $lead = null,
 
         // Check for leads using unique identifier
         if (count($uniqueFieldData)) {
-            $existingLeads = $this->getRepository()->getLeadsByUniqueFields($uniqueFieldData, ($lead) ? $lead->getId() : null);
+
+            $leadRepo = $this->getRepository('MauticLeadBundle:Lead');
```

Comment:
Please look at the PHPSTAN issue: https://travis-ci.org/mautic/mautic/jobs/383728598#L688

And fix the code style issues https://travis-ci.org/mautic/mautic/jobs/383728599#L713